src/lmql/runtime/openai_secret.py

```diff
@@ -1,51 +1,87 @@
+from typing import Tuple
 import os
 
-openai_secret = None
-openai_org = None
```

Comment:
These vars are set at the bottom of the file, so there is no need to set them here